home.css

```diff
@@ -7,21 +7,39 @@ body {
 }
 
 .name {
+    margin-top: 300px;
     text-align: center;
     font-size: 3rem;
-    /*border: 1px solid #FFF;*/
 }
 
 .title {
     text-align: center;
     font-size: 2rem;
-    /*border: 1px solid #FFF;*/
+    margin-top: 5px;
 }
 
-.page-links-container-home {
-    /* Vertically and horizontally centre div */
-    position: absolute;
-    top: 70%;
-    left: 50%;
-    transform: translate(-50%, -50%);
+.icons {
+    margin-top: 70px;
+    text-align: center; /* How to center inline elements*/
+}
+
+#home {
+    width: 80%;
+}
+
+
+/****************************
+Media Queries
+/****************************/
+
+@media screen and (max-width: 205px) {
+
+    .name {
+        font-size: 2.25rem;
+    }
+
+    .title {
+        font-size: 1.25rem;
```

Comment:
Same as before regarding decimals with rem units.

Reply:
Had a look. Seems to be common practice.